wiki/Welcome/en.md

```diff
@@ -1,148 +1,38 @@
+## Current events
 
-### Achievements
-_Full explanation: [Achievements][Achievements wikilink]_
+<!--There are currently no active events within the osu! community at this time.-->
+- [Aspire Stage Three](https://osu.ppy.sh/home/news/2017-09-21-aspire-stage-three-osucatch-begins)
+- [osu!idol 2017 (Auditions)](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open)
+- [osu!mania 4K World Cup 2017](/wiki/Tournaments/MWC/2017/4K)
+- [Yuzu's New Look Contest](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest)
 
-Achievements are little badges on one's profile which indicate that someone has accomplished an exceptional goal. Achievements' badge can be unlocked in various ways such as playing a lot or hitting a certain combo count in one beatmap. When you fulfilled the criteria, you will receive a badge at "Online Results" and your account's personal achievement tab. However, the meat of it is the hidden mission, Hush-Hush, where the criteria is not known and it is up to us to figure it out on how to get it. They are pretty hard to get but it is fulfilling when you managed to achieved it by yourself.
+## Ranking process
 
-**Note:** Achievement badges can only be unlocked online based on your current account logged-in.
+_See also: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)._
 
-## Chat
-*Full explanation: [Chat Console][Chat_Console wikilink]*
+The ranking process of a beatmap (a difficulty that the player can play) consists of [beatmapping](/wiki/beatmapping/#getting-started) and [modding](/wiki/modding/#getting-started).
 
-Knowing the basics on how the [Chat Console][Chat_Console wikilink] works is important to those who are interested into making new friends in our friendly environment. It's also important to understand how to use it in case someone messages you (unlikely to happen unless you are well-known), telling you tips or asking for help themselves. There is a button to open up the chat on the bottom right of the screen but it's way quicker to just press F8 on your keyboard.
+Beatmapping is the process where a creator creates a beatmap. This process includes: selecting music, timing and mapping the beatmap, and testing the beatmap; and could include: adding a video, [storyboarding](/wiki/storyboarding/#getting-started), and/or [skinning](/wiki/skinning/#getting-started). All of those can be done by yourself or with other users, sometimes referred to _storyboarders_, _skinners_, and/or _guest creators_, depending on what they do.
```

Comment:
replace "; and could include" with ". It might also include"
for consistency, replace "storyboarding" with just "a storyboard" (it will continue the "adding" clause)
similarly, replace "skinning" with "a custom skin"
replace "by yourself" with "by the creator alone"

Reply:
about the "storyboarding" to storyboard, I sorta did that intentionally (even if it was inconsistent) because storyboard and storyboarding are two entirely different articles 🤷‍♂️